### DISCUSSION TITLE
Change the order of tabs

### DISCUSSION THREAD

**Discussion by ronaldohoch** [GrapesJS/grapesjs#3979] [Q&A] [answered]
Hello!
Is there a Grapesjs way to change the order of the tabs?
![image](https://user-images.githubusercontent.com/2287371/143625976-6e2f14bf-7cce-4691-879b-9c1e52bb0109.png)

**Comment by Ju99ernaut** [comment, marked as answer]
I think you can try to use the css `order` property

**Reply by ronaldohoch** [reply]
Yes, i've made this way ^^
I thought there might be a way to sort on `grapesjs.init`

Thank you ^^
